apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/GetObjectOptions.cs

```diff
@@ -30,6 +30,11 @@ public sealed class GetObjectOptions
         /// </summary>
         public long? Generation { get; set; }
 
+        /// <summary>
```

Comment:
Not that we can do anything about the API, but this feels odd to me - I'd have expected an option of "include soft-deleted" rather than "*only* soft-deleted". Ah well. Maybe it's a better fit for real use cases.

Reply:
(My intuition fits with https://google.aip.dev/132 mind you...)

Reply:
The Storage team has said that the surface as is, is very intentional.

Reply:
Oh well...